openmmapi/include/PlumedForce.h

```diff
@@ -78,6 +78,15 @@ class OPENMM_EXPORT_PLUMED PlumedForce : public OpenMM::Force {
     bool usesPeriodicBoundaryConditions() const {
         return false;
     }
+    /**
+     * Set temperature. A negative value means undefined and not passed to PLUMED.
+     * By default it is set to -1.
+     */
+    void setTemperature(double KbT);
```

Comment:
Argument should be `temperature` instead of `KbT`.  While you're at it, could you also modify the description to say, "Set temperature, measured in Kelvin"?  It's good to be clear about units.

Reply:
Done!